liferay-echo/liferay-portal
# LPD-17381 Remove all &nbsp; characters by using item selector and resource selector taglib

## Motivation

[Link to ticket](https://liferay.atlassian.net/browse/LPD-15679) due to this [LPP-52603](https://liferay.atlassian.net/browse/LPP-52603)

### Changes

Use the item selector for layouts and the resource selector taglib.

### Test scenario

1. Go to the Home page
2. Add the following widget: Site Map
3. Click on ⋮ > Configuration
4. Click on the field below Root Page
5. Observe the results

https://github.com/liferay-echo/liferay-portal/assets/93203423/9fd4e714-4c0d-4a23-98ef-5467b9daa4e9
